tests/e2e/config.dev.ts

```diff
@@ -6,6 +6,8 @@ const appPath = './android/app/build/outputs/apk/development/debug/app-developme
 const config: Config = {
     MAIN_APP_PACKAGE: packageName,
     DELTA_APP_PACKAGE: packageName,
+    BRANCH_MAIN: 'main',
+    BRANCH_DELTA: 'main',
     MAIN_APP_PATH: appPath,
     DELTA_APP_PATH: appPath,
```

Comment:
this is on purpose?

Reply:
@hannojg yes. When we submit test result, then we specify `branch` and then we use the name of the branch to store the result.

`Flashlight` is running on the server, so we have to distinguish different builds (that's why I added `BRANCH_MAIN`/`BRANCH_DELTA`).

For `dev` config we run the same debug build (always `main`). That's why we specify `main` in both cases 🙂